superflore/generators/ebuild/gen_packages.py

```diff
@@ -58,7 +58,7 @@ def err(string):
 def make_dir(dirname):
     try:
         os.makedirs(dirname)
-    except:
+    except Exception:
```

Comment:
This is repeated as above. Is there somewhere it can be shared?

Reply:
Response as above ;)